anchor.js

```diff
@@ -49,7 +49,7 @@
      * @return {Boolean} - true if the current device supports touch.
      */
     this.isTouchDevice = function() {
-      return !!(('ontouchstart' in window) || window.DocumentTouch && document instanceof DocumentTouch);
+      return Boolean('ontouchstart' in window || window.DocumentTouch && document instanceof DocumentTouch);
```

Comment:
https://github.com/Modernizr/Modernizr/blob/a0bab3c7bce629990acfaa67955632eeaee9f2e1/feature-detects/touchevents.js#L41

Reply:
Yeah, [this issue](https://github.com/bryanbraun/anchorjs/issues/47) has more background on why that feature was added.

That's interesting that Modernizr removed it. I'd be open to removing it at some point but it would be a breaking API change. I think I'd rather hold off unless I specifically hear reports of people having issues with it.

Reply:
Perhaps just update it to the latest version before they removed it? And leave the removal for a major version.

Reply:
Ok, I updated this to [the last version they used before removing the feature](https://github.com/Modernizr/Modernizr/pull/2432/files#diff-cf0b19caddca681af0bb076e85025f32L40), and pushed up the commit.